### DISCUSSION TITLE
How to unistall sgpt --install-functions

### DISCUSSION THREAD

**Discussion by stanyslassz** [TheR1D/shell_gpt#454] [General]
After installer sgpt --install-functions, sgpt can work without confirmation .
How to rollback this one ?
Thx a lot for your work

**Comment by TheR1D** [comment]
Edit `~/.config/shell_gpt/.sgptrc` change line `OPENAI_USE_FUNCTIONS=false`. You can also disable functions when calling `sgpt` e.g. `sgpt "some text" --no-functions`.
